util-core/src/main/scala/com/twitter/util/Closable.scala

```diff
@@ -45,6 +45,14 @@ abstract class AbstractClosable extends Closable
  */
 object Closable {
   private[this] val logger = Logger.getLogger("")
+  private[this] var collectorThreadEnabled = true
+  
+  
+  /** Stop CollectClosables thread. */
+  def stopCollectClosablesThread() {
+	collectorThreadEnabled = false
```

Comment:
We should perhaps protect against calling this multiple times.

Reply:
OK